pages/start_page.py

```diff
@@ -0,0 +1,83 @@
+import logging
+from time import sleep
+
+from constants.start_page import StartPageConst
+from pages.base_page import BasePage
+
+
+class StartPage(BasePage):
+    """Stores methods describes start page actions"""
+
+    def __init__(self, driver):
+        super().__init__(driver)
+        self.const = StartPageConst
+        self.hello_page = HelloPage
```

Comment:
Why do you need this one ?

Reply:
Fix : deletestring 14 "self.hello_page = HelloPage"